### PR TITLE
Feature/196 update docs

### LINKED ISSUE #196: Update documentation for all components
- Add TSDoc comments everywhere
- Make sure it looks nice in Storybook

### PR DESCRIPTION
**Pull request by tobiasljungstrom**
All the comments!

Also tried to make the Docs page in SB look its best.

Closes #196

### CODE REVIEW

File: src/atoms/row/Row.tsx [hunk 1]
```diff
@@ -48,7 +59,7 @@ const StyledRow = styled.div(
 	({ alignX, alignY, reversed }: StyledRowProps) => css`
 		align-items: ${alignY};
 		display: flex;
-		flex-flow: ${reversed};
+		flex-direction: ${reversed ? 'row-reverse' : 'row'};
```
Comment (tomahl) [inline src/atoms/row/Row.tsx:62]: Is this check necessary? Flex-direction defaults to row anyway..

File: src/atoms/visually-hidden/VisuallyHidden.stories.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default {
 	},
 };
 
-const Template: Story<VisuallyHiddenProps> = () => (
+const Template: Story<HTMLAttributes<HTMLSpanElement>> = () => (
```
Comment (tomahl) [inline src/atoms/visually-hidden/VisuallyHidden.stories.tsx:16]: Why don't we use the default syntax here?

File: src/atoms/visually-hidden/VisuallyHidden.tsx [hunk 3]
```diff
@@ -2,20 +2,20 @@ import styled from '@emotion/styled';
...
+ * Renders content invisible and removes it from document flow, while keeping it accessible
+ * to screen readers, document parsing, and similar technology.
+ *
+ * @privateRemarks TODO: Add DSM link?
+ *
+ * @beta
+ */
 const VisuallyHidden = forwardRef(
-	({ children, ...rest }: VisuallyHiddenProps, ref: ForwardedRef<HTMLSpanElement>) => (
+	({ children, ...rest }: HTMLAttributes<HTMLSpanElement>, ref: ForwardedRef<HTMLSpanElement>) => (
```
Comment (tomahl) [inline src/atoms/visually-hidden/VisuallyHidden.tsx:18]: Why did you remove the interface? Now children is optional and not mandatory.

File: src/atoms/checkbox/Checkbox.tsx [hunk 4]
```diff
@@ -11,28 +11,49 @@ import Sizes from '~tokens/sizes/Sizes';
...
+ * A checkbox to be used in forms. See also 'CheckboxField'.
+ *
+ * @remarks
+ * {@link https://skf.invisionapp.com/dsm/ab-skf/4-web-applications/nav/5fa7caf78c01200018354495/asset/5ea69c129da91e75d5f73f1d | DSM Documentation}
+ *
+ * @beta
+ */
 const Checkbox = forwardRef(
 	(
 		{
```
Comment (tomahl) [inline src/atoms/checkbox/Checkbox.tsx:50]: Props doesn't appear to be sorted a-z.

Comment (tobiasljungstrom) [reply src/atoms/checkbox/Checkbox.tsx:50]: Will fix, good catch!

File: src/atoms/visually-hidden/VisuallyHidden.tsx (hunk 3 above)
Comment (tobiasljungstrom) [reply src/atoms/visually-hidden/VisuallyHidden.tsx:18]: Aha, it's to make the children mandatory of course. I missed that. I thought the interface was just repeating what it extended and was therefore unnecessary. I'll revert it.

File: src/atoms/row/Row.tsx (hunk 1 above)
Comment (tobiasljungstrom) [reply src/atoms/row/Row.tsx:62]: True, it can be made more succinct!

**Review by raggeju — APPROVED** [review]
